lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -409,8 +434,48 @@ class CanvasKitRenderer implements Renderer {
   ui.ParagraphBuilder createParagraphBuilder(ui.ParagraphStyle style) =>
     CkParagraphBuilder(style);
 
+  final Map<ui.FlutterView, _RenderQueue> _renderQueues =
+      <ui.FlutterView, _RenderQueue>{};
```

Comment:
Is there an advantage in putting the `_RenderQueue` in a map, compared to making it a field on `ViewRasterizer` (or `FlutterView`)? It seems the map requires that we handle more states than we need. For example, when nothing is rendering we can express it as `_RenderQueue.current == null`, but `Map` can also be in `_renderQueues.contains(view) == false` state, which also means the same thing. We also need to remember to remove the view after it's been disposed of (in fact, I don't see any code that does that).

This comment is non-blocking. Just trying to see if we can simplify it a bit.

Reply:
Good idea. This simplifies things quite a bit and lets us get away with not storing the view in the render request anymore, since it is already known by the view rasterizer.